pkg/web_css/lib/src/_home.scss

```diff
@@ -108,14 +108,14 @@
   .mini-list-item {
     background: var(--pub-neutral-bgColor);
     border-radius: 4px;
-    box-shadow: 0px 2px 7px 0px var(--pub-home_card-box_shadow-color);
+    box-shadow: var(--pub-box-shadow);
```

Comment:
Should use something other than `shadow` at all? just curious.

Reply:
See the linked articles: the basic suggestion is to use border + lighter background for elevating count out of its surroundings. I still find those mostly flat, and I thought the inset + minimal shadow contrast may be interesting to check out. If that is not winning here, we shall do border + lighter background (though the inset is kind of that, with slight gradient changes).